desktop/windows/troubleshoot.md

```diff
@@ -274,7 +274,7 @@ Your machine must have the following features for Docker Desktop to function cor
 
 1. Virtual Machine Platform
 2. [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10){:target="_blank" rel="noopener" class="_"}
-3. Virtualization enabled in the BIOS
+3. [Virtualization enabled in the BIOS](https://archive.ph/0m7oE){:target="_blank" rel="noopener" class="_"}
```

Comment:
Looks like this URL is a mirror of the original content (with lots of advertising / tracking).

The original content comes from https://github.com/ucberkeley/bce-documentation/blob/gh-pages/enabling-virtualization-in-your-pc-bios.md, and from that repository, it looks like the official URL for that page is https://bce.berkeley.edu/enabling-virtualization-in-your-pc-bios.html

```suggestion
3. [Virtualization enabled in the BIOS](https://bce.berkeley.edu/enabling-virtualization-in-your-pc-bios.html){:target="_blank" rel="noopener" class="_"}
```